promscale/promscale-run-queries.md

```diff
@@ -0,0 +1,245 @@
+# Run queries with Promscale
+You can use Promscale to run queries in SQL and in PromQL.
+
+SQL queries are handled directly by TimescaleDB. PromQL queries can be directed
+to the Promscale Connector. Alternatively, you can direct PromQL queries to the
+Prometheus instance, which reads data from the Connector using the `remote_read`
+interface. The Connector, in turn, fetches data from TimescaleDB.
+
+You can run SQL queries in Promscale with your preferred SQL tool. In this
+section, we use `psql`. For more information about installing and using `psql`,
+see the [installing psql section][install-psql].
+
+## Query a metric
+When you query a metric, the query is performed against the view of the metric
+you're interested in. This example queries a metric named `go_dc_duration` for
+its samples in the past five minutes. This metric is a measurement for how long
+garbage collection is taking in Go applications:
+``` sql
+SELECT * from go_gc_duration_seconds
+WHERE time > now() - INTERVAL '5 minutes';
+```
+
+An example of the output for this query:
+``` bash
+time            |    value    | series_id |      labels       | instance_id | job_id | quantile_id
```

Comment:
Headers and columns don't align properly. Presumably because the whitespace in front of `time` is missing.